app/assets/stylesheets/application.css

```diff
@@ -164,6 +164,17 @@ position: absolute;
   margin-top: 2em;
 }
 
+
 .request-neighborhood {
   margin-top: 2em;
-}
\ No newline at end of file
+}
+
+.info-article {
+  text-align: justify !important;
+}
+
+.info-box {
+  margin-bottom: 2em;
+}
+>>>>>>> Stashed changes
```

Comment:
Not sure what's going on here, but I don't think line 179 or 180 should be included.